### PR TITLE
Request Detail Page - Request Info Bar and Stage List

### PR DESCRIPTION
**Pull request by lgalis**
First phase for the Detail page for an Approval request - included Request info bar and Stage transcript.
1. Request Info Bar
2. Request Stage Transcript - list all stages 
4. Modal dialogs for Add comment, Deny and approve
5. Redux work to get al comments for all stages of a request

Work to come in the next PRs:
1. Redux/submit for actions ( add comment, submit, deny)
2. Chronological comments list in detail page
3. Detail page breadcrumbs

Screenshot from 2019-04-23 09-41-18

Screenshot from 2019-04-23 09-41-25

Screenshot from 2019-04-23 09-41-37

Screenshot from 2019-04-23 09-41-49

Screenshot from 2019-04-23 09-42-29

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -18,6 +18,14 @@
   }
 }
 
+.detail-pane {
+  background-color: white;
+  minHeight: 100%;
```
Comment (Hyperkid123) [inline src/App.scss]: this should be `min-height`

File: src/smart-components/request/request-detail/request-detail.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { withRouter, Route } from 'react-router-dom';
+import { Grid, GridItem } from '@patternfly/react-core';
+import { Section } from '@red-hat-insights/insights-frontend-components';
+import '../../../App.scss';
+import ActionModal from '../action-modal';
+import RequestInfoBar from './request-info-bar';
+import RequestStageTranscript from './request-stage-transcript';
+//import TopToolbar from '../../../presentational-components/shared/top-toolbar';
```
Comment (Hyperkid123) [inline src/smart-components/request/request-detail/request-detail.js]: left over comment?

File: src/smart-components/request/request-detail/request-detail.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+import { fetchRequest } from '../../../redux/actions/request-actions';
+import { RequestLoader } from '../../../presentational-components/shared/loader-placeholders';
+
+const RequestDetail = ({
+  match: { url },
+  //history: { push },
```
Comment (Hyperkid123) [inline src/smart-components/request/request-detail/request-detail.js]: ^^

File: src/smart-components/request/request-detail/request-detail.js [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+  if (isLoading || Object.keys(selectedRequest).length === 0) {
+    return (
+      <Section style={ { backgroundColor: 'white', minHeight: '100%' } }>
+        <RequestLoader />
+      </Section>
+    );
+  }
+  else {
+    console.log('Not loading, request: ', selectedRequest);
```
Comment (Hyperkid123) [inline src/smart-components/request/request-detail/request-detail.js]: console log

File: src/smart-components/request/request-detail/request-stage-transcript.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import React, { Fragment }  from 'react';
+import PropTypes from 'prop-types';
+import { Title } from '@patternfly/react-core';
+import StageList from './stage-list';
+
+const RequestStageTranscript = ({ request }) => {
+  console.log('Stages:', request ? request.stages : []);
+  return (<Fragment>
+    <Title size="sm" style={ { paddingLeft: '5%' } }> Stage(s) transcript </Title>
```
Comment (Hyperkid123) [inline src/smart-components/request/request-detail/request-stage-transcript.js]: I might be wrong on this but i think PF is using static spacing values (8, 16, 32 and 64)

**Comment by lgalis** [comment]
@Hyperkid123 - addressed feedback, thank you!

**Pull request merged**